### ISSUE TITLE
The Course 2 And The Succeeding Courses?

### ISSUE THREAD

**Issue by The-ML-Hero** [Zuu97/TensorFlow-Advanced-Techniques-Specialization-Coursera-deeplearning.ai#1]
Hi I just wanted to ask whether you'll be uploading the 2nd course and the succeeding courses when it gets released?
Please Reply As Soon As Possible

**Comment by machinelearningzuu** [comment]
Yes, I will upload. Now I'm doing the second course and I will upload all codes for the second course tomorrow.

**Comment by The-ML-Hero** [comment]
Thanks A Lot!

**Comment by machinelearningzuu** [comment]
First 2 courses have been uploaded. sorry for the delay.
